crates/nu-cli/src/shell/completer.rs

```diff
@@ -25,6 +32,27 @@ impl NuCompleter {
             .map(|block| crate::completion::engine::completion_location(line, &block.block, pos))
             .unwrap_or_default();
 
+        let config = config_or_empty(Tag::unknown());
```

Comment:
OK, I will add this in.

Reply:
For your information, the core team actually discussed config today. We'll definitely come back and improve all of this one day 🙂

Reply:
Hrm,

This means that shell Helper has to become aware of constructing the config, I think it may be a little off-target for this. I don't want to change too much in parts of the code im not familiar with.. 

Thought?

Reply:
But yeah, reading config on completion is not ideal to put it mildly.